avocado/core/job.py

```diff
@@ -483,7 +483,7 @@ def _run(self):
 
         if 'INTERRUPTED' in summary:
             self.exitcode |= exit_codes.AVOCADO_JOB_INTERRUPTED
-        if 'FAIL' in summary:
+        if 'FAIL' in summary and not getattr(self.args, 'relax_exit_code', False):
```

Comment:
I get the use case, but how is this different than doing something like:

```sh
#!/bin/sh -e

(avocado run /bin/false || true)
(avocado run /bin/true || true)
```

Unless I'm missing something, the code that calls Avocado (or interactive shell) should be able to ignore the status code itself.  Even in an environment with `sh -e` that should be possible.